packages/eslint-config-base/index.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
       },
     ],
     'prettier/prettier': 'error',
+    'arrow-body-style': ['error', 'as-needed'],
```

Comment:
As far as I know, it's a pert of AirBnB's JavaScript style guide but it may conflict with Prettier so the rule can be disabled by it. Could you please check Prettier's config?